### ISSUE TITLE
Boilerplates don't work with private clusters

### ISSUE THREAD

**Issue by marktani** [graphcool/prisma#1964]
# Bug Report

**Current behavior**

Initializing a boilerplate to a private cluster doesn't work currently.

**Reproduction**

* create a private cluster
* run `prisma init test` and choose `node-advanced`

You'll get this output:

```sh
Running $ prisma deploy...

? Please choose the cluster you want to deploy "yarn-dev@dev" to
Added cluster: nilan-use-2 to prisma.yml
Creating stage dev for service yarn-dev ✔
Deploying service `yarn-dev` to stage `dev` on cluster `nilan-marktanner-7ea852/nilan-use-2` 583ms

Changes:

  Post (Type)
  + Created type `Post`
  + Created field `id` of type `GraphQLID!`
  + Created field `createdAt` of type `DateTime!`
  + Created field `updatedAt` of type `DateTime!`
  + Created field `isPublished` of type `Boolean!`
  + Created field `title` of type `String!`
  + Created field `text` of type `String!`
  + Created field `author` of type `Relation!`

  User (Type)
  + Created type `User`
  + Created field `id` of type `GraphQLID!`
  + Created field `email` of type `String!`
  + Created field `password` of type `String!`
  + Created field `name` of type `String!`
  + Created field `posts` of type `[Relation!]!`
  + Created field `updatedAt` of type `DateTime!`
  + Created field `createdAt` of type `DateTime!`

  PostToUser (Relation)
  + Created relation between Post and User

Applying changes 2.0s

Hooks:
Importing seed dataset from `seed.graphql` 1.7s

Your GraphQL database endpoint is live:

  HTTP:  https://nilan-use-2_nilan-marktanner-7ea852.prisma.sh/yarn-dev/dev
  WS:    https://nilan-use-2_nilan-marktanner-7ea852.prisma.sh/yarn-dev/dev

Checking, if schema file changed 773ms
Writing database schema to `src/generated/prisma.graphql`  0ms
Running $ graphql prepare...
 ▸    Cluster nilan-use-2, that is provided in the prisma.yml could not be
 ▸    found.
 ▸


Next steps:
  1. Change directory: `cd yarn-dev`
  2. Start local server and open Playground: `yarn dev`
```

Note this part of the output:

```
Running $ graphql prepare...
 ▸    Cluster nilan-use-2, that is provided in the prisma.yml could not be
 ▸    found.
 ▸
```

**Expected behavior?**

It works.

**Comment by Huvik** [comment]
Its not related to boilerplates, its same for my project too

**Comment by timsuchanek** [comment]
Thanks for reporting! We just released `1.2.7` and updated all boilerplates, they're now working with the private clusters as expected :)
